bounties/npm/tomato/1/vulnerability.json

```diff
@@ -0,0 +1,53 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-17",
+	"AffectedVersionRange": "*",
+	"Summary": "Arbitrary Code Injection",
+	"Contributor": {
+		"Discloser": "",
+		"Fixer": ""
+	},
+	"Package": {
+		"Registry": "npm",
+		"Name": "tomato",
+		"URL": "https://www.npmjs.com/package/tomato",
+		"Downloads": "1248"
+	},
+	"CWEs": [
+		{
+			"ID": "CWE-94",
+			"Description": ""
+		}
+	],
+	"CVSS": {
+		"Version": "3.1",
+		"AV": "N",
+		"AC": "L",
+		"PR": "N",
+		"UI": "N",
+		"S": "U",
+		"C": "H",
+		"I": "H",
+		"A": "H",
+		"E": "",
+		"RL": "",
+		"RC": "",
+		"Score": "9.8"
+	},
+	"CVEs": [
+		""
+	],
+	"Repository": {
+		"URL": "https://github.com/leizongmin/tomato",
```

Comment:
The `repo` doesn't exist `anymore`, so it would be impossible propose a fix :cry: